### ISSUE TITLE
Add test_dist.py to tests run on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8556]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8556|
|Type|enhancement|
|Created|2016-07-04 08:14:13Z|
|Branch|https://github.com/twisted/twisted/tree/8556-rodrigc-dist|

test_dist.py needs to be added to the list of tests run on Python 3,
because more of the routines inside twisted/python/dist.py are
being used on Python 3.

<details><summary>Searchable metadata</summary>

```
trac-id__8556 8556
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8556_rodrigc_dist 8556-rodrigc-dist
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467620053767377 1467620053767377
changetime__1467626895767046 1467626895767046
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/288

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_2"></a>|@Lukasa set owner to @rodrigc|
|-|-|

LGTM, merge when ready.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset 72630aa241c296946a15dd53d155e9ace9323f8f
```
#!CommitTicketReference repository="" revision="72630aa241c296946a15dd53d155e9ace9323f8f"
Merge pull request #288 from twisted/8556-rodrigc-dist

Author: rodrigc
Reviewer: Lukasa
Fixes: #8556

Add test_dist.py to tests which run on Python 3
```
